namespaces/live-1.cloud-platform.service.justice.gov.uk/formbuilder-platform-test-dev/resources/user-filestore.tf

```diff
@@ -2,7 +2,7 @@
 ##################################################
 # User Filestore S3
 module "user-filestore-s3-bucket" {
-  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=3.0"
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=3.1"
```

Comment:
Can you please use the latest Module.
https://github.com/ministryofjustice/cloud-platform-terraform-s3-bucket/blob/master/example/s3.tf

Reply:
done